terra-jupyter-python/tests/smoke_test.ipynb

```diff
@@ -36,6 +36,120 @@
     "# Test cases requiring the context of a notebook "
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Test package installations\n",
+    "\n",
+    "NOTE: installing packages via `%pip` installs them into the running kernel - no kernel restart needed."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sys.path"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!env | grep PIP"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Install a package we do not anticipate already being installed on the base image"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "output = !pip3 show rich\n",
+    "print(output)  # Should show not yet installed.\n",
+    "assert(0 == output.count('Name: rich'))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%pip install rich==10.16.1"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "output = !pip3 show rich\n",
+    "print(output)  # Should show that it is now installed!\n",
+    "assert(1 == output.count('Name: rich'))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Install a package **from source** that we do not anticipate already being installed on the base image"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "output = !pip3 show docstring-parser\n",
+    "print(output)  # Should show not yet installed.\n",
+    "assert(0 == output.count('Name: docstring-parser'))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%pip install docstring-parser==0.13"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "output = !pip3 show docstring-parser\n",
+    "print(output)  # Should show that it is now installed!\n",
+    "# TODO uncomment this test after https://github.com/DataBiosphere/terra-docker/issues/285\n",
```

Comment:
Uncomment this after https://github.com/DataBiosphere/terra-docker/issues/285 to re-enable the failing test.